gpii/node_modules/flowManager/src/PSPChannel.js

```diff
@@ -84,37 +84,16 @@ fluid.defaults("gpii.pspChannel", {
         }
     },
     events: {
-        preferencesApplied: null
+        preferencesAppliedLocal: null,
```

Comment:
I don't see that there are any listeners to this event, or tests for it

Reply:
[This event is fired](https://github.com/cindyli/universal/blob/GPII-2965/gpii/node_modules/lifecycleManager/src/LifecycleManager.js#L241) once the updated preferences are applied to the local computer. At the moment it's only used to compose [the aggregate event `preferencesApplied`](https://github.com/cindyli/universal/blob/GPII-2965/gpii/node_modules/flowManager/src/PSPChannel.js#L88-L93) that indicate the completion of the handling of the updated preferences. This aggregate event has been tested in [the PSP integration test](https://github.com/cindyli/universal/blob/GPII-2965/tests/shared/PSPIntegrationTestDefs.js#L179).

Reply:
OK - could you add a comment indicating that its main current purpose is to compose the aggregate event - thanks